test/StandardAssetRegistry.js

```diff
@@ -242,6 +242,61 @@ contract('StandardAssetRegistry', accounts => {
     })
   })
 
+  describe('Transactions Related', () => {
+    it('ensures the asset is passed to another owner after transfer', async () => {
+      await registry.generate(3, CONTENT_DATA, { from: creator })
+      await registry.transfer(anotherUser, 3)
+      const newOwner = await registry.holderOf(3)
+      newOwner.should.be.equal(anotherUser)
+    })
+    it('ensures the sender owns the asset that is trying to send', async () => {
+      await registry.generate(4, CONTENT_DATA, { from: anotherUser })
+      await assertRevert( registry.transfer(operator, 4) )
+    })
+    it('after receiving an asset, the receiver can\'t transfer the sender\'s other assets', async () => {
+      await registry.generate(5, CONTENT_DATA, { from: creator })
+      await registry.transfer(user, 5)
+      await assertRevert( registry.transfer(anotherUser, 5) )
+      await assertRevert( registry.transfer(operator, 5) )
+      await assertRevert( registry.transfer(mallory, 5) )
+      const newOwner = await registry.holderOf(5)
+      newOwner.should.be.equal(user)     
+    })
+    it('throws if no arguments are sent', async () => {
+      AssertError(registry.transfer())
```

Comment:
given that it's a function, I'd go with `assertError`. Also it's in line with `assertRevert`.

Reply:
quack